oaknational/Oak-Web-Application
# [CUR-213] Add help center banner to curric visualiser

## Description

- Added help centre banner to bottom of curriculum visualiser page
- Includes button that redirects to help centre

## How to test

1. Go to https://deploy-preview-1935--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Click on any subject phase combination
3. Click on the curriculum visualiser
4. You should see the banner at the bottom of the page
5. Click the button as a link to test it redirects correctly to the help centre

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2023-09-27 at 11 49 08](https://github.com/oaknational/Oak-Web-Application/assets/32605982/74259b50-ecbf-48a0-87a0-22aa5ea6be2c)

How it should now look:
![Screenshot 2023-09-27 at 16 57 02](https://github.com/oaknational/Oak-Web-Application/assets/32605982/a7015c20-4844-4e23-b369-b574d4babbab)


## Checklist
